framework_tvb/tvb/interfaces/web/static/js/bursts.js

```diff
@@ -950,18 +950,38 @@ function displayPseSimulationMessage() {
     const THREASHOLD_WARNING = 500;
     const THREASHOLD_ERROR = 50000;
 
-    const pse_param1_lo = $("#pse_param1_lo")[0].valueAsNumber;
-    const pse_param1_hi = $("#pse_param1_hi")[0].valueAsNumber;
-    const pse_param1_step = $("#pse_param1_step")[0].valueAsNumber;
-    const pse_param1_number = calculateValuesInRage(pse_param1_lo, pse_param1_hi, pse_param1_step);
+    // Check if we have first range parameter with guid
```

Comment:
Again, this might go in a reusable function, once the IDs of the HTML elements is given as params.